test-assets/7677a9/slider.js

```diff
@@ -0,0 +1,44 @@
+function increaseSlider() {
+    document.getElementById('motionSlider').value++;
+    document.getElementById('output').innerHTML = document.getElementById('motionSlider').value;
+}
+
+function decreaseSlider() {
+    document.getElementById('motionSlider').value--;
+    document.getElementById('output').innerHTML = document.getElementById('motionSlider').value;
+}
+
+function handleOrientation(event) {
+    if (event.gamma > 20) {
+        increaseSlider();
+    } else if (event.gamma < -20) {
+        decreaseSlider();
+    }
+}
+
+function handleOrientationCanBeDisabled(event) {
+    const disableMotion = document.getElementById('disableMotion');
+    const gamma = !disableMotion.checked ? event.gamma : 0;
+
+    if (event.gamma > 20) {
+        increaseSlider();
+    } else if (event.gamma < -20) {
```

Comment:
Tests should be done on `gamma` otherwise it can't be disabled.
```suggestion
    if (gamma > 20) {
        increaseSlider();
    } else if (gamma < -20) {
```